osu.Game/Overlays/Profile/Sections/Beatmaps/PaginatedBeatmapContainer.cs

```diff
@@ -38,5 +38,27 @@ public PaginatedBeatmapContainer(BeatmapSetType type, Bindable<User> user, strin
                 Anchor = Anchor.TopCentre,
                 Origin = Anchor.TopCentre,
             };
+
+        protected override int GetCount(User user)
+        {
+            switch (type)
```

Comment:
Switch expressions can be correctly handled now. Here is the definitive use case. I turned off the suggestion, and will turn it on soon.